src/rollback_to_stable/rts.c

```diff
@@ -44,11 +44,14 @@ __wt_rts_check(WT_SESSION_IMPL *session)
     WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
     WT_RTS_COOKIE cookie;
+    uint32_t session_cnt;
```

Comment:
Please revert the changes in this file, the `txn_sessions_walked` stat is meant to be used for traversals of the `txn_shared_list` this is traversing the conn->sessions array so it shouldn't increment these stats.